ReportService.Tests/DbExporterTests.cs

```diff
@@ -0,0 +1,167 @@
+using Autofac;
+using AutoMapper;
+using Monik.Client;
+using Monik.Common;
+using Moq;
+using ReportService.Core;
+using ReportService.Entities;
+using ReportService.Entities.Dto;
+using ReportService.Extensions;
+using ReportService.Interfaces.Core;
+using ReportService.Interfaces.Operations;
+using ReportService.Interfaces.ReportTask;
+using ReportService.Operations.DataExporters;
+using ReportService.Operations.DataExporters.Configurations;
+using ReportService.Operations.DataExporters.Dependencies;
+using ReportService.ReportTask;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ReportService.Tests
+{
+
+    public class DbExporterTests
+    {
+
+        #region fields
```

Comment:
Do we really need these #region statements?

Reply:
I do like regions in code..